### PR TITLE
Fix issue 396 - Reject quotes from user name

### LINKED ISSUE #396: Usernames are being saved with quotes
Some email providers put the users name in quotes, so when a user is added through `email_processor` the user ends up getting saved with a name that includes quotes.  These should be cleaned prior to saving.

### PR DESCRIPTION
**Pull request by lccezinha**
Fix #396

Add a constant to make possible add another invalid chars for use name, and use a callback to verify if user name needs to be sanitazed before create a user.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -94,6 +96,7 @@ class User < ActiveRecord::Base
 
   after_invitation_accepted :set_role_on_invitation_accept
   after_create :enable_notifications_for_agents
+  before_create :reject_invalid_characters_from_name
   acts_as_taggable_on :teams
```
Comment (scott) [inline app/models/user.rb]: Could you change to before_save so people can't go in and add invalid characters later on?  I doubt they would, but right now they could

**Comment by lccezinha** [comment]
Hello @scott I will fix merge error, and I'll accept you suggestion to change callback and push it again soon.

**Comment by lccezinha** [comment]
Hello @scott can you see my changes again ?

**Comment by scott** [comment]
Sorry for the delay, I have been on vacation out of the country.  I think this looks good, let me double check it and merge it in

**Comment by lccezinha** [comment]
Ok @scott, Thanks!

**Pull request merged**